react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergBridgeJS2Parent.java

```diff
@@ -81,6 +84,28 @@ public static MediaType getEnum(String value) {
         }
     }
 
+    enum GutenbergUserEvent {
+        PAGE_TEMPLATE_APPLIED("page_template_applied");
```

Comment:
I went through and made some changes thanks to the clarification in https://github.com/wordpress-mobile/gutenberg-mobile/issues/1883#issuecomment-590247803 I renamed this event to `editor_session_template_apply`

Reply:
Thanks, I'll make a similar change. :+1: